pkg/actors/actors.go

```diff
@@ -270,26 +288,27 @@ func (a *actorsRuntime) Init(ctx context.Context) error {
 	return nil
 }
 
-func (a *actorsRuntime) getAppHealthCheckChan(ctx context.Context) <-chan bool {
+func (a *actorsRuntime) getAppHealthChecker() (*health.Checker, error) {
 	if len(a.actorsConfig.Config.HostedActorTypes.ListActorTypes()) == 0 || a.appChannel == nil {
-		return nil
+		return nil, nil
 	}
 
 	// Be careful to configure healthz endpoint option. If app healthz returns unhealthy status, Dapr will
 	// disconnect from placement to remove the node from consistent hashing ring.
 	// i.e if app is busy state, the healthz status would be flaky, which leads to frequent
 	// actor rebalancing. It will impact the entire service.
-	return a.getAppHealthCheckChanWithOptions(ctx,
+	return a.getAppHealthCheckerWithOptions(
 		health.WithFailureThreshold(4),
-		health.WithInterval(5*time.Second),
+		health.WithHealthyStateInterval(5*time.Second),
+		health.WithUnHealthyStateInterval(time.Second/2),
```

Comment:
Sometimes the apps could become un-healthy when they are overloaded. In this case, I think that an interval of .5s can add to an overloaded app too much?

```suggestion
		health.WithUnHealthyStateInterval(1500*time.Millisecond),
```

Reply:
> I think that an interval of .5s can add to an overloaded app too much?

I don't think so- 500ms is a really long time for a  HTTP endpoint that isn't doing any work. We have the request timeout to handle overloaded networks.